specification/portal/resource-manager/Microsoft.Portal/preview/2015-08-01-preview/portal.json

```diff
@@ -440,9 +440,7 @@
         "metadata": {
           "description": "The dashboard part's metadata.",
           "type": "object",
-          "additionalProperties": {
-            "type": "object"
-          }
+          "additionalProperties": true
```

Comment:
>"additionalProperties": true [](start = 10, length = 28)

What is expected as additionalProperties?

You should not allow any additionalProperties if possible. If you need to pass additionalProperties it should be a strongly typed object, this is required by Azure Policy as well, and this applies to all your other resource types.

Reply:
The challenge in case of Azure dashboards is that the parts are defined by extensions and the part schema isn't standard. We don't control the schema on the RP side. The idea behind "additionalProperties" is to allow all the different schemas that exist for different parts.

Reply:
@jorgecotillo What would be the recommendation for dashboards considering our scenarios?

Reply:
@msHich @jorgecotillo Do you have an ETA on this? We are tracking this in https://github.com/microsoft/vscode-azurearmtools/issues/1146

Reply:
@jianyexi @jorgecotillo  Any recommendations for dashboard specific scenario since we dont have a definition of all the parts available on the RP side?

Reply:
@jianyexi @jorgecotillo can you please help move this forward?

Reply:
Usually we don't recommend adding free-style additionalProperties as it could have an impact on other services such as Policy (some teams might want to enforce a Policy on a non-defined property and it is not possible to do so), but given this is a metadata, it should be ok. [Here](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi032-use-of-additionalproperties---always-define-strongly-typed-objects-and-avoid-free-style-objects) it is our reference

Reply:
Thanks @jorgecotillo.